flake8_pie/tests/test_pie805_prefer_literal.py

```diff
@@ -0,0 +1,44 @@
+from __future__ import annotations
+
+import ast
+
+import pytest
+
+from flake8_pie import Flake8PieCheck
+from flake8_pie.pie805_prefer_literal import PIE805
+from flake8_pie.tests.utils import Error, ex, to_errors
+
+EXAMPLES = [
+    ex(
+        code="""
+"foo".encode()
+""",
+        errors=[PIE805(lineno=2, col_offset=0)],
+    ),
+    ex(
+        code="""
+"foo".encode("utf-8")
+""",
+        errors=[PIE805(lineno=2, col_offset=0)],
+    ),
+    ex(
+        code="""
+"foo".encode("utf8")
+""",
+        errors=[PIE805(lineno=2, col_offset=0)],
+    ),
+    ex(
+        code="""
+b"foo"
+"foo".encode("ascii")
+"foo".encode("bar")
+""",
+        errors=[],
```

Comment:
What should happen if you have a non-ascii character in your string?

```
assert "😀".encode() == b'\xf0\x9f\x98\x80'
```

Reply:
oh interesting, we also need to check that it only contains ascii literal chars